packages/ckeditor5-utils/src/version.ts

```diff
@@ -24,138 +24,162 @@ declare global {
-	 *
-	 * # Confused an editor build with an editor implementation
-	 *
-	 * This scenario is very similar to the previous one, but has a different origin.
-	 *
-	 * Let's assume that you wanted to use CKEditor 5 from source.
+	 * <details>
+	 * <summary>(Legacy) Predefined builds</summary>
 	 *
-	 * The correct way to do so is to import an editor and plugins and run them together like this:
+	 * If you are using the {@glink getting-started/legacy/installation-methods/predefined-builds Predefined builds},
+	 * you cannot import any additional plugins. This is because the predefined builds already include the editor core
+	 * and selected plugins. Importing additional plugins will cause some modules to be bundled and loaded twice.
 	 *
-	 * ```ts
-	 * import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
-	 * import Essentials from '@ckeditor/ckeditor5-essentials/src/essentials';
-	 * import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
-	 * import Bold from '@ckeditor/ckeditor5-basic-styles/src/bold';
-	 * import Italic from '@ckeditor/ckeditor5-basic-styles/src/italic';
+	 * Examples:
 	 *
-	 * ClassicEditor
-	 * 	.create( document.querySelector( '#editor' ), {
-	 * 		plugins: [ Essentials, Paragraph, Bold, Italic ],
-	 * 		toolbar: [ 'bold', 'italic' ]
-	 * 	} )
-	 * 	.then( editor => {
-	 * 		console.log( 'Editor was initialized', editor );
-	 * 	} )
-	 * 	.catch( error => {
-	 * 		console.error( error.stack );
-	 * 	} );
+	 * ```js
+	 * import ClassicEditor from '@ckeditor/ckeditor5-build-classic'; // ✅
+	 * import { Highlight } from 'ckeditor5'; // ❌
+	 * import { Highlight } from '@ckeditor/ckeditor5-highlight/dist/index.js'; // ❌
+	 * import { Highlight } from '@ckeditor/ckeditor5-highlight'; // ❌
+	 * import Highlight from '@ckeditor/ckeditor5-highlight/src/highlight'; // ❌
+	 * import '@ckeditor/ckeditor5-highlight/build/highlight'; // ❌
 	 * ```
 	 *
-	 * However, you might have mistakenly imported a build instead of the source `ClassicEditor`. In this case
-	 * your imports will look like this:
-	 *
-	 * ```ts
-	 * import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
-	 * import Essentials from '@ckeditor/ckeditor5-essentials/src/essentials';
-	 * import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
-	 * import Bold from '@ckeditor/ckeditor5-basic-styles/src/bold';
-	 * import Italic from '@ckeditor/ckeditor5-basic-styles/src/italic';
+	 * If you are missing some features from the
+	 * {@glink getting-started/legacy/installation-methods/predefined-builds#plugins-included-in-the-predefined-builds list of plugins},
+	 * you should switch to the {@glink updating/nim-migration/migration-to-new-installation-methods new installation methods}
+	 * which do not have this limitation.
+	 * If you cannot migrate to the new installation methods, you can try the
+	 * {@glink getting-started/legacy/installation-methods/predefined-builds#superbuild superbuild} instead,
```

Comment:
Is it still available?

Reply:
Because all predefined builds were removed from CDN at the same time, the developers can migrate from one to another as long as they stay on the same CKEditor version.